.travis.yml

```diff
@@ -21,8 +25,12 @@ install:
     - mkdir files
 
 script:
-    - python setup.py test -a "--verbose"
-    - find . -type f -name "*.py" ! -name 'conf.py' ! -name 'ugrid_wx.py' | xargs flake8 --max-line-length=100
+    - if [[ $TEST_TARGET == 'default' ]]; then
+        python setup.py test -a "--verbose";
+      fi
+    - if [[ $TEST_TARGET == 'coding' ]]; then
+        find . -type f -name "*.py" ! -name 'conf.py' ! -name 'ugrid_wx.py' | xargs flake8 --max-line-length=100;
```

Comment:
I think that #114 was fixed. So we can remove `ugrid_wx.py` from the ignore list and show what this PR can do here :wink: